docs: translate blogs on 08/2021 (#680)
iresty/iresty.github.io: website/blog/2021/08/24/shanghai-meetup-recap.md

---
title: Apache APISIX Meetup in Shanghai
author: Apache APISIX
keywords:
- API Gateway
- APISIX
- Meetup
description: "Event Review: Apache APISIX Meetup Shanghai Topic Sharing and Video Review."
tags: [Events]
---

> Event review: Apache APISIX Meetup Shanghai topic sharing and video review.

> <! --truncate-->

Reviewer: ```suggestion
<!--truncate-->
```

> <! --truncate-->

Reviewer: ```suggestion
<!--truncate-->
```

> Apache APISIX Meetup Shanghai was successfully held last Saturday (August 21) 🎉🎉🎉🎉

Reviewer: ```suggestion
Apache APISIX Meetup Shanghai was successfully held last Saturday (August 21st) 🎉🎉🎉🎉
```

> The Meetup was initiated by [API7.AI](https://www.apiseven.com/zh), and invited partners such as Akiyon, AirWallex, and other partners to come together with Apache APISIX technical experts, Apache APISIX PMC members, and community technical experts to discuss about The conference was a great opportunity to discuss Apache APISIX community development, industry practices and other topics.

Reviewer: ```suggestion
The Meetup was initiated by [API7.AI](https://www.apiseven.com/zh), and invited partners such as iQiYi, AirWallex, and other partners to come together with Apache APISIX technical experts, Apache APISIX PMC members, and community technical experts to discuss about The conference was a great opportunity to discuss Apache APISIX community development, industry practices and other topics.
```

Let's take a look back at some of the most memorable talks from the conference!

> ## Apache APISIX-based Aiki API Gateway Implementation Practice

Reviewer: ```suggestion
## Apache APISIX-based iQiYi API Gateway Implementation Practice
```

> **Instructor**: He Cong

Reviewer: ```suggestion
**Instructor**: Cong He
```

> **Personal Introduction**: Senior R&D Engineer, IIG Infrastructure Department - Compute Cloud, mainly responsible for Aikii Gateway development and operation and maintenance.

Reviewer: ```suggestion
**Personal Introduction**: Senior R&D Engineer, IIG Infrastructure Department - Compute Cloud, mainly responsible for iQiYi Gateway development and operation and maintenance.
```

> **Personal Introduction**: Senior R&D Engineer, IIG Infrastructure Department - Compute Cloud, mainly responsible for iQiyi Gateway development and operation and maintenance.

Reviewer: ```suggestion
**Personal Introduction**: Senior R&D Engineer, IIG Infrastructure Department - Compute Cloud, mainly responsible for iQiYi Gateway development and operation and maintenance.
```

> **Detail of Topic**: API gateway has become an indispensable part of the microservice architecture. It bears the sole gateway for services to the outside world, but also needs to meet the common functions of many applications. As an online video company, Aikiy has to carry tens of millions of calls every day, and has extremely strict requirements for data security, user request response time, and system stability, so the company needs to build a high-performance, highly available API gateway. This sharing briefly introduces the architecture of Aiki API gateway and the functions implemented, including request distribution, conditional routing, Api management, flow restriction and fusion, security triple prevention, monitoring and alarming, and full link tracking.

Reviewer: ```suggestion
**Detail of Topic**: API gateway has become an indispensable part of the microservice architecture. It bears the sole gateway for services to the outside world, but also needs to meet the common functions of many applications. As an online video company, iQiYi has to carry tens of millions of calls every day, and has extremely strict requirements for data security, user request response time, and system stability, so the company needs to build a high-performance, highly available API gateway. This sharing briefly introduces the architecture of iQiYi API gateway and the functions implemented, including request distribution, conditional routing, Api management, flow restriction and fusion, security triple prevention, monitoring and alarming, and full link tracking.
```

**Video Review**: [Akiyoshi's Apache APISIX-based API Gateway Implementation Practice](https://www.bilibili.com/video/BV1Qq4y1M7bK)

## How to use Apache APISIX for centralized authentication

**Instructor**: Xinxin Zhu

**Personal Introduction**: API7 Engineer, Apache APISIX Committer

**Detail of Topic**: Authentication is a very important part of zero-trust architecture. Authentication can effectively protect services from unauthorized malicious access, data leakage, and hacker attacks. APISIX, as a dynamic, real-time, high-performance API gateway, supports rich authentication plugins, and this sharing will introduce the benefits of centralized authentication and how to perform centralized authentication on APISIX.

**Video Review**: [Using API Gateway APISIX for Centralized Authentication](https://www.bilibili.com/video/BV1WA411c7pa)

## Data Sovereignty and Apache APISIX Gateway Landing Practices

**Instructor**: Yang Li

> **Personal Introduction**: Head of AirWallex AirCloudHub Technology Platform

Reviewer: ```suggestion
**Personal Introduction**: Head of AirWallex Technology Platform
```

**Detail of Topic**: Data is everywhere, and massive amounts of data are processed, transmitted, and stored all over the world every moment. However, the world of data is not a place outside the law, the collection and processing of data must comply with the laws of each country. The API gateway has a special position in the enterprise architecture and therefore plays a special role in the data sovereignty scenario.

**Video Review**: [Apache APISIX and Data Sovereignty in Practice](https://www.bilibili.com/video/BV1GL4y1Y7sR)

## The one who travels alone travels fast, the one who travels with many travels far, Apache APISIX is grateful to have you

> **Instructor**: Wang Yansheng

Reviewer: ```suggestion
**Instructor**: Yuansheng Wang
```
